### DISCUSSION TITLE
Hyperion using homeassistant lights

### DISCUSSION THREAD

**Discussion by jgeorge1983** [hyperion-project/hyperion.ng#1454] [Ideas]
I have recently moved all my zigbee lights from Hue bridge to my zigbee dongle, and it got me thinking, is their a way to add my homeassisntant lights into hyperion?
Its not one of the network options I see, and I wondered if anyone had any experience of this?

**Comment by Lord-Grey** [comment]
I suggest you check with the wider community on the Hyperion forum:

https://hyperion-project.org/forum/

**Reply by jgeorge1983** [reply]
Thanks, have done!

**Comment by kuba-orlik** [comment]
You might be interested in this adapter: https://community.home-assistant.io/t/use-home-assistant-lights-with-hyperion/678249
